### PR TITLE
Add feature user can rate albumns and songs

### LINKED ISSUE #63: Add functionality so the user can rate songs and albums

### PR DESCRIPTION
**Pull request by condef5**
closed #63

### CODE REVIEW

File: app/controllers/admin/songs_controller.rb [hunk 1]
```diff
@@ -15,8 +15,11 @@ def new
 
     def create
       @song = Song.new(song_params)
-      @song.save
-      redirect_to admin_song_path(@song), notice: "The song was successfully created"
+      if @song.save(song_params)
+        redirect_to admin_song_path(@song), notice: "The song was successfully created"
+      else
+        render :new
```
Comment (luciagirasoles) [inline app/controllers/admin/songs_controller.rb]: how is this related to the feature?

File: app/controllers/albums_controller.rb [hunk 2]
```diff
@@ -8,4 +8,22 @@ def index
   def show
     @album = Album.find(params[:id])
   end
+
+  def rating
+    @album = Album.find(params[:id])
+    if !@album.ratings.where(user_id: current_user.id).any?
+      @album.ratings.create(value: params[:rating], user_id: current_user.id)
+      flash[:notice] = "Added to favorite albums" if params[:rating].to_i == 1
```
Comment (luciagirasoles) [inline app/controllers/albums_controller.rb]: why if you rate an album is this added to favorites?

File: app/controllers/admin/songs_controller.rb (hunk 1 above)
Comment (condef5) [reply app/controllers/admin/songs_controller.rb]: The validations are missing when create a new song in admin dashboard.

File: app/controllers/albums_controller.rb (hunk 2 above)
Comment (condef5) [reply app/controllers/albums_controller.rb]: it is just a message that notifies the user when he gives a positive rating

Comment (luciagirasoles) [reply app/controllers/albums_controller.rb]: so, why not another kind of message? such as...good choice in case of positive or we are gonna improve it for you if its a negative rating

Comment (condef5) [reply app/controllers/albums_controller.rb]: `We are gonna improve it for you if its a negative rating`. IMHO I think the rating depends on the user's taste, if you do not like an album, we as a platform would not have to question your decision, but if it is a positive action we show a message as the YouTube platform does.

Comment (luciagirasoles) [reply app/controllers/albums_controller.rb]: whatever is the case, what does it has to do with something like a "favorite list"?
